src/main/java/com/tchristofferson/configupdater/ConfigUpdater.java

```diff
@@ -147,12 +147,24 @@ private static void write(FileConfiguration defaultConfig, FileConfiguration cur
 
         String currentIgnoredSection = null;
         String line;
-        while ((line = reader.readLine()) != null) {
+        lineLoop : while ((line = reader.readLine()) != null) {
             String trimmedLine = line.trim();
 
-            if (trimmedLine.isEmpty() || trimmedLine.startsWith("#") || trimmedLine.startsWith("-"))
+            if (trimmedLine.isEmpty() || trimmedLine.startsWith("#"))
                 continue;
 
+	        if(trimmedLine.startsWith("-")) {
+		        for(String ignoredSection : ignoredSections) {
+		            boolean isIgnoredParent = ignoredSection.equals(keyBuilder.toString());
+		
+		            if(isIgnoredParent || keyBuilder.isSubKeyOf(ignoredSection)) {
+                        valueBuilder.append("\n" + line);
+	            	    
+		                continue lineLoop;
+                    }
+		        }
```

Comment:
This poor bracket is so lonely way out here  🤣

Reply:
lol, that has been modified now.